Common/TableProducer/trackextension.cxx

```diff
@@ -48,7 +48,7 @@ constexpr long run3grp_timestamp = (1619781650000 + 1619781529000) / 2;
 const char* ccdbpath_lut = "GLO/Param/MatLUT";
 const char* ccdbpath_geo = "GLO/Config/Geometry";
 const char* ccdbpath_grp = "GLO/GRP/GRP";
-const char* ccdburl = "https://alice-ccdb.cern.ch"; /* test  "http://alice-ccdb.cern.ch:8080"; */
+const char* ccdburl = "http://alice-ccdb.cern.ch"; /* test  "http://alice-ccdb.cern.ch:8080"; */
```

Comment:
maybe 8080 still needs to be added

Reply:
I think 8080 points to the test instance...

Reply:
If you don't put `https` then you have to put `8080` which is the test endpoint

Reply:
I don't think so
http://alice-ccdb.cern.ch/browse/GLO/GRP/GRP --> production
http://alice-ccdb.cern.ch:8080/browse/GLO/GRP/GRP --> test

Reply:
Yeap!
You're right!
Sorry for the noise!